lib/team.rb

```diff
@@ -0,0 +1,12 @@
+class Team
+  attr_reader :id, :franchise_id, :name, :abbreviation, :stadium, :link
```

Comment:
Depending on what the team decides we may what these in multiple lines.

Reply:
@guardiac Thanks! I changed these to multi-line to be consistent with Saryn's Game class but can adjust if the team decides otherwise.